packages/edit-widgets/src/components/widget-area/index.js

```diff
@@ -47,19 +47,57 @@ function getBlockEditorSettings( blockEditorSettings, hasUploadPermissions ) {
 }
 
 function WidgetArea( {
+	id,
 	blockEditorSettings,
-	blocks,
-	hasUploadPermissions,
 	initialOpen,
 	isSelectedArea,
 	onBlockSelected,
-	updateBlocks,
-	widgetAreaName,
 } ) {
+	const { blocks, widgetAreaName, hasUploadPermissions, rawContent } = useSelect(
+		( select ) => {
+			const {
+				canUser,
+				getEditedEntityRecord,
+			} = select( 'core' );
+			const widgetArea = getEditedEntityRecord( 'root', 'widgetArea', id );
+			return {
+				blocks: widgetArea && widgetArea.blocks,
+				rawContent: get( widgetArea, [ 'content' ], '' ),
```

Comment:
`widgetArea.content` will be an object with a `.raw` property before the first edit.